tox.ini

```diff
@@ -24,12 +24,13 @@ deps =
     lockfile==0.10.2
     coverage
     mock==1.0.1
-    django14: Django==1.4.22
```

Comment:
We were deliberately using specific version numbers here. Having inexact version numbers could cause a lot of confusion in the case of a regression which could be caused by either a new commit to django-mailer, or a change in the latest version of Django 1.X.Y. It gets worse when a local tox environment might be using an old point release, so the developers won't easily reproduce what happens on Travis or for another user.